augur/clades.py

```diff
@@ -147,16 +144,40 @@ def assign_clades(clade_designations, all_muts, tree, ref=None):
         sorted_nodes = sorted(node_counts, key=lambda x: x.leaf_count, reverse=True)
         if len(sorted_nodes) > 0:
             target_node = sorted_nodes[0]
-            clade_membership[target_node.name] = {'clade_annotation': clade_name, 'clade_membership': clade_name}
+            basal_clade_nodes[target_node.name] = clade_name
+            clade_membership[target_node.name] = clade_name # basal nodes are members of the clade
 
     # third pass to propagate 'clade_membership'
     # don't propagate if encountering 'clade_annotation'
     for node in tree.find_clades(order = 'preorder'):
         for child in node:
-            if 'clade_annotation' not in clade_membership[child.name]:
-                clade_membership[child.name]['clade_membership'] = clade_membership[node.name]['clade_membership']
+            # if the child doesn't define the start of its own clade, but the parent belongs to a clade, then inherit that membership
+            if child.name not in basal_clade_nodes and node.name in clade_membership:
+                clade_membership[child.name] = clade_membership[node.name]
+    return (basal_clade_nodes, clade_membership)
+
 
-    return clade_membership
+def create_node_data_structure(basal_clade_nodes, clade_membership, args):
+    node_data = {}
+
+    if (not args.label_name and not args.trait_name):
+        print("WARNING: running `augur clades` without specifying --label-name and/or")
+        print("         --trait-name is deprecated. To preserve backwards compatibility")
+        print("         we will use 'clade_annotation' and 'clade_membership', respectively.")
+        print("         Note that 'clade_annotation' is now exported as a 'branch_label'")
+
+        label_name = "clade_annotation"
```

Comment:
Upon review I think this should be `clade`.

While `augur clades` used to export a node-trait `clade_annotation`, `augur export v2` would turn this into a branch label called `clade`. Thus, we should export a branch label `clade` here as `augur export v2` will leave this unchanged.